docs/dev/installing.md

```diff
@@ -175,16 +175,18 @@ The following commands run the Image Classification Code Sample using the [`dog.
 Linux and macOS:
 
 ```sh
-cd ~/inference_engine_cpp_samples_build/intel64/Release
+cd ~/openvino_cpp_samples_build/<architecture>/Release
 ./classification_sample_async -i ~/Downloads/dog.bmp -m ~/ir/googlenet-v1.xml -d CPU
 ```
+where the <architecture> is the output of ``uname -m``, for example, ``intel64``, ``armhf``, or ``aarch64``.
 
 Windows:
 
 ```bat
-cd  %USERPROFILE%\Documents\Intel\OpenVINO\inference_engine_samples_build\intel64\Release
+cd  %USERPROFILE%\Documents\Intel\OpenVINO\openvino_cpp_samples_build\<architecture>\Release
 .\classification_sample_async.exe -i %USERPROFILE%\Downloads\dog.bmp -m %USERPROFILE%\Documents\ir\googlenet-v1.xml -d CPU
 ```
+where the <architecture> is the output of ``uname -m``, for example, ``intel64``, ``armhf``, or ``aarch64``.
```

Comment:
Windows does not have uname command

I suggest providing words description for windows: `where the <architecture> is either intel64 or aarch64 depending on platform architecture`